fcrepo-audit-triplestore/src/main/java/org/fcrepo/camel/audit/triplestore/AuditSparqlProcessor.java

```diff
@@ -106,6 +107,7 @@ public void process(final Exchange exchange) throws Exception {
     private static final Property RDF_TYPE = createProperty(RDF + "type");
 
     private static final String EMPTY_STRING = "";
+    private static final String RESOURCE_TYPES = "org.fcrepo.jms.resourceType";
```

Comment:
See my earlier comment. In the next version of fcrepo-camel, I would expect the `JmsHeaders` class to be completely removed, as the code should begin to rely on the body serialization defined in the draft SPI spec (message headers play no role in that spec)